server/remote_cache/digest/digest.go

```diff
@@ -637,3 +637,15 @@ func (g *Generator) RandomDigestBuf(sizeBytes int64) (*repb.Digest, []byte, erro
 	}
 	return d, buf, nil
 }
+
+// ParseFunction parses a digest function name to a proto.
+func ParseFunction(value string) (repb.DigestFunction_Value, error) {
+	switch value {
+	case "SHA256", "sha256", "":
+		return repb.DigestFunction_SHA256, nil
+	case "BLAKE3", "blake3":
+		return repb.DigestFunction_BLAKE3, nil
+	default:
+		return 0, status.InvalidArgumentErrorf("unknown digest function %q", value)
+	}
```

Comment:
I think this can be:

```
		if df, ok := repb.DigestFunction_Value_value[strings.ToUpper(dfString)]; ok {
			digestFunction = repb.DigestFunction_Value(df)
		} else {
			return nil, status.InvalidArgumentErrorf("Unknown digest function: %q", dfString)
		}
```

then you can use this function on line 435 where we do this today

Reply:
Done